app/dashboard/helpers.py

```diff
@@ -369,8 +367,7 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
 
         for old_bounty in old_bounties:
             if old_bounty.current_bounty:
-                submissions_comment_id = old_bounty.submissions_comment
```

Comment:
pushed up a migration for the old fields we are not using anymore...

though i'm not sure this is to spec... we are massively consolidating the commenet ids were tracking?

Reply:
This may have just been a first iteration spec for code review.. @mbeacom @PixelantDesign any thoughts?